src/Compilers/CSharp/Test/Emit/Emit/CompilationEmitTests.cs

```diff
@@ -397,6 +397,34 @@ public void EmitMetadataOnly_DisallowOutputtingNetModule()
         }
 
         [Fact]
+        public void EmitMetadata_DisallowEmbedding()
```

Comment:
(only if we are emitting metadata only, without the full assembly being produced we should report error).